lib/tag-list/index.tsx

```diff
@@ -116,7 +128,7 @@ export class TagList extends Component<Props> {
   reorderTag: SortEndHandler = ({ newIndex, nodes, oldIndex }) => {
     const tagName = nodes[oldIndex].node.dataset.tagName;
 
-    this.props.reorderTag(tagName, newIndex);
+    this.props.reorderTag(tagName, newIndex + 1);
```

Comment:
What was broken before that `+1` fixes?

Reply:
It was giving us the correct value. Our reorder logic was flawed. I fixed out logic.

Reply:
https://github.com/Automattic/simplenote-electron/pull/2302/files#diff-159d5dbfee67d319843d07f814f1e2115598b5b40e076aef780b73e09c1cadf3R417